src/content/docs/game-design/godot/dungeoncrawler/0-scenesetup/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+type: tutorial
+unitTitle: Making a 2D top-down dungeon crawler
+title: Setting up our scene
+description: This page works through 2D top-down dungeon crawler step-by-step
+sidebar:
+    order: 2
+---
+
+import Checklist from '/src/components/tutorial/Checklist.astro';
+import Box from '/src/components/tutorial/Box.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+This is a guide to making a 2-dimensional dungeon crawling game in [Godot](https://godotengine.org/). If you are unfamiliar with Godot, check out the [Godot basics](/game-design/godot/basics) doc as this tutorial assumes basic knowledge of navigating and using the Godot Engine.
+
+:::note[Version]
+This guide is up-to-date with Godot 4.3 stable official release but will most likely work with any 4.x release.
```

Comment:
Are tile map layers available pre Godot 4.3? Should we update it to say any future 4.x.